scripts/extract_timings.py

```diff
@@ -0,0 +1,39 @@
+#! /usr/bin/env python
+# Time-stamp: <2016-05-31 14:58:27 chrplr>
+#
+
+""" Extracts slice timing information from a dicom file.
+To be passed  as argument to slice_order for slice timing correction algorithms """
+
```

Comment:
Hi,

I have put a dataset at:

https://www.nitrc.org/frs/download.php/9035/epi-example-dicom.tar.gz

As soon as spm12 works in the master branch of pypreprocess, I will test it
on this dataset.

I still need to write code to fetch the dataset...

Chris

--
Christophe Pallier

Contact info: http://www.pallier.org/w/pmwiki.php/Main/Contact

On Sun, Jun 19, 2016 at 1:18 PM, DOHMATOB Elvis <notifications@github.com>
wrote:

> In scripts/extract_timings.py
> <https://github.com/neurospin/pypreprocess/pull/231#discussion_r67615123>:
>
> > @@ -0,0 +1,39 @@
> > +#! /usr/bin/env python
> > +# Time-stamp: <2016-05-31 14:58:27 chrplr>
> > +#
> > +
> > +""" Extracts slice timing information from a dicom file.
> > +To be passed  as argument to slice_order for slice timing correction algorithms """
> > +
>
> If I remember good, we were considering hosting a small dataset (say on
> NTRC), that could be used here to make this demo self-contained. Any update
> ?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/neurospin/pypreprocess/pull/231/files/8caa896e8ea92a2478829bd93dbdbe53d287f26a#r67615123>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ABdbMvWtLial8myl397edxDu0Il1GHwSks5qNSWAgaJpZM4IqgU1>
> .
>